src/gtasa/effects/custom/ped/PedsCanExplodeOnDieEffect.cpp

```diff
@@ -0,0 +1,51 @@
+#include "util/CPedDamageResponseCalculator.h"
+#include "util/EffectBase.h"
+#include "util/GenericUtil.h"
+#include "util/hooks/HookMacros.h"
+
+#include <extensions/ScriptCommands.h>
+
+using namespace plugin;
+
+static unsigned int randomValue;

```

Comment:
Please include the static variable within the effect class like it's done in other effects